participants/abatten/simple_functions.py

```diff
@@ -10,3 +10,15 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_a_prime(value):
+
+    max_quotient = math.ceil(math.sqrt(value))
+
+    for i in range(max_quotient):
```

Comment:
One efficiency improvement you could make here is to skip even numbers using `range`s step functionality: range(1, max_quotient, 2).